lib/fluent/plugin/in_groonga.rb

```diff
@@ -155,6 +155,12 @@ def start
             run
           end
         end
+        if respond_to?(:detach_multi_process)
+          $log.warn("[input][groonga][warn] detach_multi_process is not supported in this Fluentd version. ignored.")
```

Comment:
I think that dependency should specified in gemspec instead of applying this patch like as https://github.com/fluent/fluentd/blob/3519b20c9be772c4da10a7bc599eb7c3c81cd103/fluentd.gemspec#L21.